Dockerfile

```diff
@@ -3,6 +3,7 @@
 FROM python:alpine AS base
 WORKDIR /app
 COPY requirements.txt .
+RUN apk add --no-cache g++
```

Comment:
sorry, I don't understand the intention of this line. I guess it is there to prevent caching of another language – correct?